### PR TITLE
Add four new cookbooks

### PR DESCRIPTION
**Pull request by Jerimee**
#### Summary
```SUMMARY: Content "Add four new cookbooks"```

#### Purpose of change

This adds 4 new cookbooks to cooking.json, and adds them to some loot tables.

#### Describe the solution

Book entries added to data/json/items/book/cooking.json

loot tables modified at:
1. SUS_cooking_bookcase in itemgroups/SUS/library.json
2. kitchen item group in itemgroups/item_groups.json - NOTE: also reduced distribution of cookbook_italian so as not to overweight cookbooks in general
3. homebooks item group in  itemgroups/books.json
4.. added SUS_cooking_bookcase to kitchen_counters in itemgroups/collections_domestic.json - this means an entire counter might be dedicated to cookbooks (which is a real enough occurrence)
5. kitchen_nonfood in itemgroups/mansion.json

Recipes added to books at:
1. a number of places in recipe_food.json 
2. spaghetti_pesto in pasta.json (cookbook_foodfashions)
3. offal_dishes (cookbook_liverforkids)
#### Describe alternatives you've considered

The plurals I&#039;ve used do *not* have the extended name of the book, as there are easily identified shorter names. So, for example, one book is called &quot;Dainty Dishes Fit for a King,&quot; and the plural is &quot;copies of Dainty Dishes.&quot; Alternatively, this could be &quot;copies of Dainty Dishes Fit for a King.&quot; I prefer the shorter version for the sake of the UI.

I considered having these cookbooks not contain recipes for the sake of simplicity, and I considered that it might be desirable for cookbooks to have a larger number of recipes.
#### Testing

Testing done locally via wishing for the items and reading the books in game.

Referenced the doc at JSON_INFO.md#books, JSON_INFO.md#recipes, JSON_INHERITANCE.md, ITEM_SPAWN.md
#### Additional context

These books do *not* solve any problem nor are the part of any larger design. They are merely additional clutter content for variety. Note that some of the books contain recipes, but the recipes themselves are not new.

The books are:

* Liver-Licious Recipes Your Kids Will Love (cookbook_liverforkids)
* Food Fashions for Young Moderns (cookbook_foodfashions)
* Dainty Dishes Fit for a King (cookbook_daintydishes)
* Eat Your Way to a Fit Physique (cookbook_eatyrway)

For each of the recipes associated with the books I added a third display entry to reflect the theme of the particular cookbook. These display entries did not seem to work. See #36790

### CODE REVIEW

**Comment by Inglonias** [comment]
It looks like this PR doesn't actually shuffle around any existing recipes, but simply adds more sources for some that are already in place.

I thought part of the plan was to make it so that it took more than a few textbooks for a character to master a skill?

This may not be a problem, but it is something I'm confused about.

**Comment by Jerimee** [comment]
@Inglonias 

>  Note that some of the books contain recipes, but the recipes themselves are not new.

I didn't add any new recipes, I only used existing ones.

> I considered having these cookbooks not contain recipes for the sake of simplicity...

This PR is only to add greater variety to the world (ie clutter). Now "Bon Appetit" and "Cucina Italiana" won't be the only cookbooks you regularly come across.

**Comment by ymber** [comment]
The names of these are weird. Think what you would actually call it if you were publishing.

**Comment by Jerimee** [comment]
@ymber supposed to be funny, but actually most of them are based off real books! See https://www.pinterest.com/jerimee/books/cookbooks/

**Comment by ymber** [comment]
If they're real go for it. I think 60%-70% overrepresents those kind of old books in the itemgroup though.

File: data/json/items/book/cooking.json [hunk 1]
```diff
@@ -441,5 +441,62 @@
     "intelligence": 12,
     "time": "35 m",
     "fun": -1
+  },
+  {
+    "id": "cookbook_liverforkids",
+    "type": "BOOK",
+    "name": { "str": "Liver-Licious Recipes Your Kids Will Love", "str_pl": "copies of Liver-Licious Recipes" },
+    "description": "This cook book is written by liver enthusiast Tamsyn Beckerleg.  Inside you'll find a great deal of anecdotes about the near certain ruination of children who refuse to eat their liver.",
```
Comment (matskuman5) [inline data/json/items/book/cooking.json]:
```suggestion
    "description": "This cookbook is written by liver enthusiast Tamsyn Beckerleg.  Inside you'll find a great deal of anecdotes about the near-certain ruination of children who refuse to eat their liver.",
```

File: data/json/recipes/food/offal_dishes.json [hunk 2]
```diff
@@ -75,7 +75,12 @@
     "difficulty": 2,
     "time": "30 m",
     "batch_time_factors": [ 83, 5 ],
-    "book_learn": [ [ "scots_cookbook", 2 ], [ "family_cookbook", 0 ], [ "cookbook", 0 ] ],
+    "book_learn": [
+      [ "scots_cookbook", 2 ],
+      [ "family_cookbook", 0 ],
```
Comment (I-am-Erk) [inline data/json/recipes/food/offal_dishes.json]:
```suggestion
```

File: data/json/recipes/food/offal_dishes.json [hunk 3]
```diff
@@ -55,7 +55,7 @@
     "difficulty": 6,
     "charges": 1,
     "time": "30 m",
-    "book_learn": [ [ "mag_cooking", 4 ], [ "mag_glam", 5 ] ],
+    "book_learn": [ [ "mag_cooking", 4 ], [ "mag_glam", 5 ], [ "cookbook_liverforkids", 5, "Buttery Duck Bites" ] ],
```
Comment (I-am-Erk) [inline data/json/recipes/food/offal_dishes.json]:
```suggestion
    "book_learn": [ [ "mag_glam", 5 ], [ "cookbook_liverforkids", 5, "Buttery Duck Bites" ] ],
```

File: data/json/recipes/recipe_food.json [hunk 4]
```diff
@@ -649,7 +649,7 @@
     "subcategory": "CSC_FOOD_MEAT",
     "skill_used": "cooking",
     "difficulty": 4,
-    "book_learn": [ [ "cookbook_sushi", 4 ] ],
+    "book_learn": [ [ "cookbook_sushi", 4 ], [ "cookbook_foodfashions", 6, "Rice-Free Sashimi Donburi" ] ],
```
[the remark this answers is not in the document]
Comment (Jerimee) [reply data/json/recipes/recipe_food.json]: This was supposed to be a joke - like asking a bartender for a white russian, hold the vodka and without kahlua - you get a glass of milk

**Comment by Jerimee** [comment]
Yeah, eatyrway doesn't have any recipes, just skill training. 

Maybe I could add 

1. sugar_from_wood
2. pizza_veggy
3. dahlia_baked
4. soup_cullenskink

File: data/json/recipes/recipe_food.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (I-am-Erk) [reply data/json/recipes/recipe_food.json]: It's still the same joke, but also joking about current food fashion. If you make a low carb dish out of a bowl of rice and fish, you get fish.

**Comment by I-am-Erk** [comment]
I'd go with protein shake and things like that

File: data/json/items/book/cooking.json [hunk 5]
```diff
@@ -441,5 +441,62 @@
...
+    "required_level": 1,
+    "max_level": 2,
+    "intelligence": 5,
+    "time": "45 m"
+  },
+  {
+    "id": "cookbook_daintydishes",
+    "type": "BOOK",
+    "name": { "str": "Dainty Dishes Fit for a King", "str_pl": "copies of Dainty Dishes" },
+    "description": "This cookbook contains many fanciful recipes. Some of them are even worth the trouble to actually prepare. Lovely pen and ink illustrations of overweight knights and gluttonous monks line the pages.",
```
Comment (Maleclypse) [inline data/json/items/book/cooking.json]: ` "description": "This cookbook contains many fanciful recipes.  Some of them are even worth the trouble to actually prepare.  Lovely pen and ink illustrations of overweight knights and gluttonous monks line the pages.",`

File: data/json/items/book/cooking.json [hunk 6]
```diff
@@ -441,5 +441,62 @@
...
+    "max_level": 1,
+    "time": "70 m",
+    "fun": -1,
+    "copy-from": "cookbook_liverforkids"
+  },
+  {
+    "id": "cookbook_foodfashions",
+    "type": "BOOK",
+    "name": { "str": "Food Fashions for Young Moderns", "str_pl": "copies of Food Fashions" },
+    "description": "A cookbook professing itself to be for those seeking \"daring and sophisticated flavors.\"  The book exhorts the reader to defy convention and break the rules, and then presents a number of useful rules and conventions for preparing food.",
```
Comment (Maleclypse) [inline data/json/items/book/cooking.json]: `\"daring and sophisticated flavors.\"`
Can quotation marks be used inside the description?

Comment (I-am-Erk) [reply data/json/items/book/cooking.json]: Sure, if they're broken like that.

**Comment by Jerimee** [comment]
> I'd go with protein shake and things like that

Yeah, that's what I planned to do, but most of the recipes are autolearn: true (including protein shake)

File: data/json/items/book/cooking.json (hunk 5 above)
Comment (Jerimee) [reply data/json/items/book/cooking.json]: Isn't that what I wrote?

Comment (I-am-Erk) [inline data/json/items/book/cooking.json]:
```suggestion
    "description": "This cookbook contains many fanciful recipes. Some of them are even worth the trouble to actually prepare. Lovely pen and ink illustrations of overweight knights and gluttonous monks line the pages.",
```

**Comment by I-am-Erk** [comment]
I would suggest putting Tofu on the list for the boring book, at least. More could be considered later. I think someone needs to do an audit of what's on autolearn, honestly, and remove some stuff like hard cheese and several other very non-intuitive recipes.

**Comment by Jerimee** [comment]
TODO: put tofu (line 541), tofu_fried (555) and pizza_veggy (3505) on the eatyrway (boring) book.

I'll do this tonight.

**Pull request merged**